distributed/_concurrent_futures_thread.py

```diff
@@ -156,6 +182,9 @@ def weakref_cb(_, q=self._work_queue):
             self._threads.add(t)
             _threads_queues[t] = self._work_queue
 
+    def interrupt(self, ident):
+        _async_raise(ident)
```

Comment:
functionally no difference but for readability, I would prefer `WorkerThreadInterrupt` being mentioned here. If the `_async_raise` is renamed that may also be fine but at this point we rely specifically to raise this exception and not something else